ui/raidboss/data/06-ew/trial/sophia-un.ts

```diff
@@ -0,0 +1,719 @@
+      // Instead, we warn the user when Barbelo separates from Sophia, which is 7D7D.
+      id: 'SophiaUN Light Dew',
+      type: 'Ability',
+      netRegex: { id: '7D7D', source: 'Sophia', capture: false },
+      condition: (data) => data.clonesActive,
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Avoid head laser',
+          de: 'Kopflaser ausweichen',
+          fr: 'Évitez le laser de la tête',
+          ja: 'レーザーを避ける',
+          cn: '躲避人头炮',
+          ko: '머리 레이저 피하기',
+        },
+      },
+    },
+    {
+      id: 'SophiaUN Execute',
+      type: 'StartsUsing',
+      netRegex: { id: '7DA4', source: 'Sophia' },
+      durationSeconds: (_data, matches) => parseFloat(matches.castTime),
+      alertText: (data, _matches, output) => {
+        const localeCompass: { [dir: string]: string } = {
+          'NW': output.northwest!(),
+          'NE': output.northeast!(),
+          'SW': output.southwest!(),
+          'SE': output.southeast!(),
+        };
+        const safeSpots = [];
+        for (const dir of Object.keys(data.isQuadrantSafe)) {
+          if (data.isQuadrantSafe[dir])
+            safeSpots.push(localeCompass[dir]);
+        }
+        if (safeSpots.length === 1) {
+          return safeSpots[0];
+        }
+        return output.multiple!({
+          dir1: safeSpots[0],
+          dir2: safeSpots[1],
+        });
+      },
+      outputStrings: {
+        northwest: Outputs.northwest,
+        northeast: Outputs.northeast,
+        southwest: Outputs.southwest,
+        southeast: Outputs.southeast,
+        multiple: {
+          en: '${dir1} / ${dir2}',
+          de: '${dir1} / ${dir2}',
+          fr: '${dir1} / ${dir2}',
+          ja: '${dir1} / ${dir2}',
+          cn: '${dir1} / ${dir2}',
+          ko: '${dir1} / ${dir2}',
+        },
+      },
+    },
+    {
+      id: 'SophiaUN Clone Cleanup',
+      type: 'Ability',
+      netRegex: { id: '7DA4', source: 'Sophia', capture: false },
+      delaySeconds: 5,
+      run: (data) => {
+        delete data.clonesActive;
+        delete data.seenThunder;
+        data.isQuadrantSafe = { NW: true, NE: true, SW: true, SE: true };
+      },
+    },
+    {
+      // The eight Sophia entities on the scale pans have IDs that run sequentially,
+      // from n + 0 to n + 7. They spawn into the instance in a random order, but the
+      // locations where they spawn have a fixed association with their sequence number offset:
+
+      // 0: (-55.0637, -3.496415)
+      // 1: (-55.06367, -10.1404)
+      // 2: (-55.06363, 9.5766)
+      // 3: (-55.0637, 3.523648)
+      // 4: (54.9907, 3.387837)
+      // 5: (54.98699, 9.5779B3)
+      // 6: (54.9907, -3.50686)
+      // 7: (54.99068, -10.14043)
+
+      // Because of this, we need only see one entity use a 21 log line and we can find the rest.
+      id: 'SophiaUN Quasar Setup',
+      type: 'Ability',
+      netRegex: { id: '19A[89]' },
```

Comment:
This is probably the issue, if I had to guess? This line looks unchanged.

Reply:
Looks like it, yes. Definitely a good reason to avoid the compact form in the future!